### ISSUE TITLE
Installation from pip on a clean system is broken

### ISSUE THREAD

**Issue by karpov-sv** [esheldon/esutil#73]
When installing on a clean system using `pip install esutil` the installation goes fine, but the package cannot be imported. 
The error is:
```
root@836b9e6d53c4:/# python3 -c 'import esutil'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File "/usr/local/lib/python3.10/dist-packages/esutil/__init__.py", line 104, in <module>
    from . import integrate
ImportError: cannot import name 'integrate' from partially initialized module 'esutil' (most likely due to a circular import) (/usr/local/lib/python3.10/dist-packages/esutil/__init__.py)
```

Steps to replicate (using Ubuntu docker image as an example, just to have a clean system - but it manifests on other distributions as well):

```
docker run -it ubuntu
> apt update
> apt install python3-pip
> pip3 install esutil
> python3 -c 'import esutil'
```

**Comment by esheldon** [comment]
does this apt installed python include numpy?

**Comment by esheldon** [comment]
Note this also works, using the ubuntu system python3 rather than conda
```
python3 -m venv test-esutil
pip3 install numpy
pip3 install esutil
python3 -c 'import esutil'
```

**Comment by karpov-sv** [comment]
No, of course not.
So most probably the reason is what esutil does not list numpy as its dependency (it is commented out in setup.py?..)

This really breaks things when installing esutil as a requirement for another package - it tries to pull everything including numpy (which my package also depends on), and then install esutil before (or in parallel?..) with numpy, and thus produces broken wheel - that's how I actually encountered it.

**Comment by esheldon** [comment]
good point.  Do you want to take a shot at a PR?

**Comment by jobovy** [comment]
I believe this was fixed by #77.

**Comment by aboucaud** [comment]
Hi, I just encountered this issue when it installing `esutil` from pip
```shell
python -m pip install esutil
``` 
which installs a precompiled v0.6.10.

After reading this thread, I tried to install the development version instead
```
python -m pip install git+https://github.com/esheldon/esutil.git
```
and it solved the issue.

Since 0.6.10 is dated from Jan 8 and #77 was merged on Jan 10, I also believe this has been fixed in the current version. Perhaps a new release on PyPI would be useful.
